release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -296,6 +296,9 @@ Code Freeze will typically fall around Weeks 8 or 9 depending on the length or r
 ### Week 13
 
 - Release Retrospective participation
+  - Make sure to confirm a facilitator for the Release Retrospective with SIG PM
+  - Make sure to replicate the meeting invite with
+    kubernetes-release-team@googlegroups.com.
```

Comment:
Ambiguous: "...replicate the meeting invite with..."

Are you saying to, eg:  "Make sure to invite kubernetes-release-team@googlegroups.com when scheduling the retrospective."

Since ContribEx has moved The Community Meeting to monthly, if that forum is to still be used, the retro would probably on average fall a bit later than week 13.  If the retro will no longer happen in The Community Meeting, we need to think about how to manage the invite for broad attendance and advertise it on k-dev and the SIG chairs list.

Reply:
There should be no replication of the invite. The following should be invited:
- Kubernetes Release Calendar
- Community Calendar
- Release Team ML
- Release Managers ML
- SIG Release ML

In addition to the broad announcement as Tim mentioned to k-dev and the SIG leads list.
Scheduling-wise, I think we should still aim to have it as close to the end of the cycle as possible, so information/experiences are still fresh.

We could probably just ask to have a special community meeting for this?

Reply:
I'm +1 on scheduling an out of band meeting for the retros. It adds 4 additional meetings per year won't throw off the schedule for the community groups to give their update during the community meeting
/cc @castrojo 
for comment on out of band community meetings.

Reply:
+1 to Bob's comment.

---

> The following should be invited:
> 
> Kubernetes Release Calendar
> Community Calendar
> Release Team ML
> Release Managers ML
> SIG Release ML

Release lead only has access to the release team calendar.
The community calendar invite should be set when scheduling with sig contribex, no?
All other calendars are "out of reach" for the lead, which is the topic of the handbook.
Still wanna add them?